translations/general.pot

```diff
@@ -41,7 +42,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
-"POT-Creation-Date: 2018-10-15 12:07+0000\n"
+"POT-Creation-Date: 2018-10-25 13:07+0200\n"
 "PO-Revision-Date: YYYY-mm-DD HH:MM+ZZZZ\n"
 "Last-Translator: NAME <EMAIL@ADDRESS>\n"
```

Comment:
Best in POEdit eens je naam en email instellen bij instellingen:

<img width="578" alt="poedit" src="https://user-images.githubusercontent.com/1823998/47549968-f72c4800-d8fd-11e8-98b6-2695dbfee65e.png">